fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -647,7 +683,7 @@ private fun BottomDrawer(
             open = !drawerState.isClosed,
             onClose = onDismiss,
             fraction = {
-                if (drawerState.anchors.isEmpty()
+                if (drawerState.anchors.isEmpty() || skipOpenState
```

Comment:
Add if for hasExpandedState, hasOpenState to invoke calculate fraction respectively. If both of them false then return 0 to show invisible scrim.

Reply:
I have added:
if( empty anchors)
0.dp
else if ( skipOpenState) calculate with EXPANDED
     else calcualte with OPEN